1.9/administration/logging/aggregating/logging-api.md

```diff
@@ -0,0 +1,132 @@
+---
+post_title: Logging API 
+menu_order: 1
+---
+
+# dcos-go/dcos-log - HTTP server for journal events
+
+# REST API
+#### Endpoints:
+- `/range/` get a range of logs matching the request query.
+- `/range/framework/{framework_id}/executor/{executor_id}/container/{container_id}` get a range of application logs.
+- `/stream/` tail logs keeping the connection opened, implements Server Sent Events.
+- `/stream/framework/{framework_id}/executor/{executor_id}/container/{container_id}` tail application logs.
+- `/fields/<field>` returns all possible unique values for a specific `<field>`.
+
+#### Request Header Accept
+- `text/plain`, `text/html`, `*/*` request logs in text format, ending with `\n`.
+- `application/json` request logs in JSON format.
+- `text/event-stream` request logs in Server-Sent-Events format.
+
+#### Request Header Last-Event-ID
+If `Last-Event-ID` is set dcos-log will use it as a cursor position. `Last-Event-ID` header works with `/stream/` endpoints only.
+
+NOTE: Accept header `text/event-stream` cannot be used with `/fields/<field>` endpoint.
+
+#### Response Header X-Journal-Skip-Prev
```

Comment:
please exclude `Response Header X-Journal-Skip-Prev` it isn't used and will be deprecated